### PR TITLE
fix trends router bug

### LINKED ISSUE #300: fix trends breaking when switching fields

### PR DESCRIPTION
**Pull request by howardchung**
fixes #300 

RCA:
Trends had no router component (withRouter).

Fix:
* Move routing mutation logic to ButtonGarden

### CODE REVIEW

**Comment by howardchung** [comment]
It was already using withRouter, so I thought it was OK.

We CAN make the parent component pass the router, but that's doubling the number of withRouter calls.

**Comment by mike-robertson** [comment]
I think what I mean here is why would a button component only manipulate urls on button clicks. I'd say this component shouldn't be aware of what clicking does as much as possible. So maybe add a component in top of this called routingButtonGarden which does this router change inside. Idk, just seems weird that a button knows so much about what clicking it does.

**Comment by mike-robertson** [comment]
Tldr: this feels like a presentational component, not such a self aware component.

**Comment by howardchung** [comment]
Integrated into #297
